packages/provider-elasticsearch/src/example-types/metricGroups/percentilesGroupStats.js

```diff
@@ -6,7 +6,7 @@ import { groupStats } from './groupStatUtils.js'
 // [1, 2, 3] -> [{to: 1}, {from: 1, to: 2}, {from: 2, to: 3}, {from: 3}]
 let boundariesToRanges = _.flow(
   F.mapIndexed((to, i, list) => F.compactObject({ from: list[i - 1], to })),
-  (arr) => F.push({ from: _.last(arr).to }, arr)
+  (arr) => F.pushOn(arr, { from: _.last(arr).to })
```

Comment:
I think I actually prefer `F.push` here. Mixing non-mutating with mutating in the same flow is kind of awkward for me. If you want to mutate an array with `push` you should just loop over the array and do that.